katcp/resource.py

```diff
@@ -595,10 +595,15 @@ def parent_name(self):
 
     @property
     def name(self):
-        """Name of this KATCP resource"""
+        """Name of this KATCPSensor"""
         return self._name
 
     @property
+    def python_id(self):
```

Comment:
Hmm - no specific preference. But may be "normalised_name" to make it less python specific. Or what about something very different like "sensor_handle" or "sensor_unih" for "unique handle". Or is it just not good to introduce such new terminology now?